app/models/attachment.js

```diff
@@ -219,9 +219,29 @@ export function addModel(dbAdapter) {
       this.fileExtension = supportedImageTypes[this.mimeType]
       this.noThumbnail = '1' // this may be overriden below
 
-      // Store original image size
       let originalImage = promisifyAll(gm(tmpAttachmentFile))
```

Comment:
Could we move it inside "else" clause(s) to avoid doing this twice for rotated images? (This one looks costly.)